lib/ex_doc/markdown/cmark.ex

```diff
@@ -24,6 +40,9 @@ defmodule ExDoc.Markdown.Cmark do
   Generate HTML output. Cmark takes no options.
   """
   def to_html(text, _opts) do
-    text |> Cmark.to_html()  |> ExDoc.Markdown.pretty_codeblocks()
+    text
+    |> Cmark.to_html()
+    |> ExDoc.Markdown.pretty_codeblocks()
+    |> ExDoc.Highlighter.highlight_code_blocks()
```

Comment:
One uses `code_blocks` and the other uses `codeblocks`.

Reply:
Although I don't think we need `pretty_codeblocks` anymore. The functionality in there should be handled by the highlighter.

Reply:
> Although I don't think we need pretty_codeblocks anymore. The functionality in there should be handled by the highlighter.

We don't and it should. It will probably be even more performant if I handle it in the same regex match.

Reply:
And regarding codeblocks vs code_blocks I prefer code_blocks. Is that ok with you?

Reply:
I would actually say none? `ExDoc.Highlighter.highlight_html`? or `ExDoc.Highlighter.highlight_html_code`?